db/migrate/20200728090010_remove_nomis_event_ids_from_moves.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveNomisEventIdsFromMoves < ActiveRecord::Migration[6.0]
+  def up
+    remove_column :moves, :nomis_event_ids
+  end
+
+  def down
+    add_column :moves, :nomis_event_ids, :integer, array: true, null: false, default: [], index: true
```

Comment:
Should we be removing an index in the up method?

Reply:
```suggestion
    add_column :moves, :nomis_event_ids, :integer, array: true, null: false, default: []
```

Reply:
Good catch, updated.